angular/core/components/angular_feedback_card/angular_feedback_card.coffee

```diff
@@ -4,7 +4,7 @@ angular.module('loomioApp').directive 'angularFeedbackCard', ->
   replace: true
   templateUrl: 'generated/components/angular_feedback_card/angular_feedback_card.html'
   controller: ($scope, AppConfig) ->
-    $scope.feedbackUrl = "https://docs.google.com/a/enspiral.com/forms/d/1JS75ZdoBiA3k7IzxDPijV4kPh1fXrmWs9A8k_WG_pyE/viewform"
+    $scope.feedbackUrl = "https://github.com/loomio/loomio/blob/master/CONTRIBUTING.md"
```

Comment:
Depending on how things go this might have to go here as well:
https://github.com/loomio/loomio_org/blob/master/components/angular_feedback_card/angular_feedback_card.coffee
(ie in the loomio.org plugin)